client/my-sites/domains/domain-search/index.jsx

```diff
@@ -42,6 +42,7 @@ import { getSuggestionsVendor } from 'lib/domains/suggestions';
  * Style dependencies
  */
 import './style.scss';
+import NewDomainsRedirectionNoticeUpsell from '../domain-management/components/domain/new-domains-redirection-notice-upsell';
```

Comment:
Another tiny thing - tis should not be in the "Style dependencies" block. Also I wonder if relative imports are good/bad thing (though I acknowledge I did something similar in the domain status code)

Reply:
I blame PHPStorm :oldmanyellsatsky: